pkg/model/result_step.go

```diff
@@ -7,25 +7,26 @@ import (
 
 type ResultStep struct {
 	// Common result for every step types
-	StepType StepType
-	StepTime time.Duration
+	StepType StepType      `json:"step_type"`
+	StepTime time.Duration `json:"step_time,omitempty"`
 
 	// Specific for type request
-	Request         rest.Request
-	Response        Response
-	Assertion       []ResultAssertion
-	VariableApplied []ResultVariable
-	VariableCreated []ResultVariable
+	Request          rest.Request      `json:"request,omitempty"`
+	Response         Response          `json:"response,omitempty"`
+	Assertions       []ResultAssertion `json:"assertions,omitempty"`
+	VariablesApplied []ResultVariable  `json:"variables_applied,omitempty"`
```

Comment:
I think that's better to use `camelCased` for json keys (property name).
variablesApplied
what do you think @thomaspoignant ?

Reply:
I have used snake case because the input file is in snake case too (to be compatible with runscope syntax).